vertical-pod-autoscaler/templates/admission-controller-deployment.yaml

```diff
@@ -21,24 +21,22 @@ spec:
         - name: admission-controller
           image: "{{ .Values.admissionController.image.repository }}:{{ .Values.admissionController.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.admissionController.image.pullPolicy }}
-          args:
-          {{- if .Values.admissionController.tlsCertArgs }}
-            {{- toYaml .Values.admissionController.tlsCertArgs | nindent 12 }}
-          {{- end }}
           env:
             - name: NAMESPACE
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
           volumeMounts:
-            {{- toYaml .Values.admissionController.volumeMounts | nindent 12 }}
+            - mountPath: /etc/tls-certs
+              name: tls-certs
+              readOnly: true
```

Comment:
I don't know why the admissionController would ever need volumeMounts, but it might be better to leave the option to add more volumeMounts in the template by leaving this line there:
`{{- toYaml .Values.admissionController.volumeMounts | nindent 12 }}`

Reply:
Added in latest commit